src/custom/sdk/custom_instance.h

```diff
@@ -73,7 +74,20 @@ class CustomInstance {
   /// output (see CustomGetOutputFn_t).
```

Comment:
Add
"\return Error code indicating success or the type of failure"
Instead of returning -1 (which is an non-defined error code) you should add a predefined error code to error_codes.h and return that . The description for that error code should be something about how a V1 version of a function was called when custom backend is not V1.